lib/lumberjack/beats/server.rb

```diff
@@ -76,8 +67,8 @@ def run(&block)
         Thread.new(connection) do |connection|
           begin
             connection.run(&block)
-          rescue Lumberjack::Beats::Connection::ConnectionClosed 
-            # Connection will raise a wrapped exception upstream, 
+          rescue Lumberjack::Beats::Connection::ConnectionClosed => e
```

Comment:
(Low priority, style) Any reason to capture the exception here when we aren't using it?